src/doc/dev_manual/ContributingHostProfiles.rst

```diff
@@ -0,0 +1,16 @@
+Contributing Host Profiles
+==========================
+
+Host profiles live in /src/resources/hosts/<compute center name>.
+
+There are a couple of helper file and scripts in the 'hosts' directory that 
```

Comment:
This sentence reads a bit odd. Maybe "There are a couple of helper files and scripts" or "There is a helper file and script" depending on the number of helper files and scripts.